models/DataObject/Localizedfield.php

```diff
@@ -462,7 +462,9 @@ private function loadLazyField(Model\DataObject\ClassDefinition\Data $fieldDefin
      * @param string|null $language
      * @param bool $ignoreFallbackLanguage
      *
-     * @return mixed
+     * @return mixed|null
+     *
+     * @throws \Exception
```

Comment:
```suggestion
     * @throws Model\Exception\NotFoundException
```

Reply:
wdyt?
![image](https://user-images.githubusercontent.com/5137917/154250287-708c63bd-782b-4110-83e9-8ad86970c593.png)

Reply:
I want to know which exceptions can be thrown. From wich line came the \Exception? Should this be catched? You can additional add `@throws \Exception` also if not. But it would be good to have only named exceptions. So I can catch it individuel... We can also add a comment when which exception is thrown.

Reply:
Let's add additional `@throws \Exception` for now.